libs/agent-sm/agent/src/test/java/org/opensearch/javaagent/AgentTestCase.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.javaagent;
+
+import org.opensearch.javaagent.bootstrap.AgentPolicy;
+import org.junit.BeforeClass;
+
+import java.security.Policy;
+import java.util.Set;
+
+public abstract class AgentTestCase {
+    @SuppressWarnings("removal")
+    @BeforeClass
+    public static void setUp() {
+        AgentPolicy.setPolicy(new Policy() {
```

Comment:
AgentPolicy.setPolicy has to be called only once, I think with inheritance it will not work

Reply:
I see now the intent is for all tests to be in `AgentTests`, huh?